src/Commands/ClearCacheFiles.php

```diff
@@ -0,0 +1,128 @@
+<?php
+/**
+ * 2020-present Friends of Presta community
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the MIT License
+ * that is bundled with this package in the file LICENSE
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/MIT
+ *
+ * @author    Friends of Presta community
+ * @copyright 2020-present Friends of Presta community
+ * @license   https://opensource.org/licenses/MIT MIT
+ */
+
+namespace FOP\Console\Commands;
+
+use FOP\Console\Command;
+use PrestaShop\PrestaShop\Adapter\Debug\DebugMode as DebugAdapter;
+use RuntimeException;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\EventDispatcher\EventDispatcher;
+use Symfony\Component\Process\Process;
+
+/**
+ * This command replace the cache directory with an empty one.
+ */
+final class ClearCacheFiles extends Command
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('fop:clear-cache')
+            ->setDescription('Replace the cache directory with an empty one.')
+            ->setHelp('This command allows you to quickly remove the cache files.');
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $io = new SymfonyStyle($input, $output);
+
+        try {
+            // first : disable event dispatcher otherwise errors will happen
+            $this->getApplication()->setDispatcher(new EventDispatcher());
+            $this->processChecks();
+            $this->deleteOldCacheDirectory(); // may exist if this command failed before
+            $this->renameCurrentCacheDirectory();
+            $this->createNewCacheDirectory(); // probably not needed
+            $io->success('New Empty cache directory created. Old cache directory deleted.');
+
+            $this->deleteOldCacheDirectory();
+            $io->text('Old directory deleted');
+
+            return 0;
+        } catch (RuntimeException $exception) {
+            $io->error("Error processing {$this->getName()}:\u{a0}" . $exception->getMessage());
+
+            return 1;
+        }
+    }
+
+    private function processChecks(): void
+    {
+        $cache_directory = $this->getCacheDirectoryBasePath();
+        if (!is_writable($cache_directory)) {
+            throw new RuntimeException("Cache directory not writable : [$cache_directory]");
+        }
+    }
+
+    private function renameCurrentCacheDirectory()
+    {
+        $process = new Process(['mv', $this->getCacheDirectoryBasePath(), $this->getCacheDirectoryOldPath()]);
```

Comment:
As discussed on slack.
That's the point of the command : bypass php files processing and use os file system directly.
Mostly because php can't delete a directory if it's not empty. So files are processed one by one on a loop, which is probably not very efficient compared to what direct file system is.

Reply:
ye well IMO, if you want performance you will not use a command to delete files, you will just delete them like any other files (in your console or explorer for windows).
IMO loosing the cross platform is a too big problem for the targeted users.
Let's wait for others opinions.

Reply:
This command offer a faster alternative to native prestashop clear-cache command.
To rely on a console command offers more trust than user typed command, it eliminates the possibility to mistype a command.

The cross plateform compat will be impleted later, that's a need for sure.

Reply:
ye ok make sens then (sorry i didn't know the whole context)

Reply:
My bad, I did not explain the need of this command ;)